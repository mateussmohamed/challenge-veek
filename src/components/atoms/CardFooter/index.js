import styled from 'styled-components'

const CardFooter = styled.footer`
  padding: 20px;
  display: flex;
  align-items: center;
  justify-content: space-between;
  flex-direction: row;
`

export default CardFooter
