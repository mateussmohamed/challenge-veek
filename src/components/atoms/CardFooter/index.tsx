import styled from 'styled-components'

export const CardFooter = styled.footer`
  padding: 20px;
  display: flex;
  align-items: center;
  justify-content: space-between;
  flex-direction: row;
`
