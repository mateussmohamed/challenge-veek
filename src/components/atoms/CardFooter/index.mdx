---
name: CardFooter
menu: Atoms
---

import { Playground, PropsTable } from 'docz'
import CardFooter from './index'

# CardFooter
<PropsTable of={CardFooter} />

<Playground>
  <CardFooter />
</Playground>
