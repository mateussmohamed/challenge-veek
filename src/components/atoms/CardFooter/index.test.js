import React from 'react'
import { shallow } from 'enzyme'
import 'jest-styled-components'

import CardFooter from './index'

describe('[<CardFooter />]', () => {
  const wrap = (props = {}) => shallow(<CardFooter {...props} />)

  test('matches to snapshot', () => {
    const wrapper = wrap({ children: 'I am a container!' })

    expect(wrapper).toMatchSnapshot()
  })
})
