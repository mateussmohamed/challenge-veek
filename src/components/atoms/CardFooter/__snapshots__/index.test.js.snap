// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[<CardFooter />] matches to snapshot 1`] = `
<StyledComponent
  forwardedClass={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": undefined,
      "componentStyle": ComponentStyle {
        "componentId": "CardFooter-sc-1c65rd-0",
        "isStatic": true,
        "rules": Array [
          "padding:20px;display:flex;align-items:center;justify-content:space-between;flex-direction:row;",
        ],
      },
      "displayName": "CardFooter",
      "render": [Function],
      "styledComponentId": "CardFooter-sc-1c65rd-0",
      "target": "footer",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
>
  I am a container!
</StyledComponent>
`;
